.github/ionic-issue-bot.yml

```diff
@@ -7,7 +7,7 @@ closeAndLock:
     - label: "ionitron: support"
       message: >
         Thanks for the issue! This issue appears to be a support request. We use this issue tracker exclusively for
-        bug reports and feature requests. Please use our [slack channel](https://stencil-worldwide.slack.com/)
+        bug reports and feature requests. Please use our [discord channel](https://ionic.link/discord)
```

Comment:
Should we use https://chat.stenciljs.com/ instead, in order to bring people directly to the Stencil General channel on the server? Or is there something special about that link that makes it such that we should avoid it? We've been using https://chat.stenciljs.com/ in some prior replacements of Slack->Discord, which is why I ask

Reply:
Updated 👍 they both go to the same destination (the root of the discord server).

Reply:
Ah, got it. Thx for the clarification!